tests/integration/high_availability/test_self_healing.py

```diff
@@ -14,19 +14,23 @@
     clean_up_database_and_table,
     ensure_all_units_continuous_writes_incrementing,
```

Comment:
Not the fault of this PR, but `ensure_all_units_continuous_writes_incrementing` on line 161 and 261 are not awaiting.

Reply:
@carlcsaposs-canonical noticed this as well and has a [separate PR](https://github.com/canonical/mysql-k8s-operator/pull/122) to address this. @dragomirp please feel free to review it if you have a moment so carl can proceed with merging it